sim/src/main/cc/endpoints/tracerv.cc

```diff
@@ -5,6 +5,9 @@
 #include <stdio.h>
 #include <string.h>
 #include <limits.h>
+#include <sstream>
+#include <iostream>
+#include <fstream>
```

Comment:
Do we need this now that you moved the test comparison to be external using scala?

Reply:
Good point.